processors/base/config.py

```diff
@@ -10,14 +10,21 @@
 from dotenv import load_dotenv
 load_dotenv('.env')
 
+# Environment
+
+ENV = os.environ.get('PYTHON_ENV', 'development')
+if os.environ.get('CI'):
+    ENV = 'testing'
 
 # Storage
 
-WAREHOUSE_URL = os.environ['WAREHOUSE_URL']
-DATABASE_URL = os.environ['DATABASE_URL']
-EXPLORERDB_URL = os.environ['EXPLORERDB_URL']
-TEST_WAREHOUSE_URL = os.environ.get('TEST_WAREHOUSE_URL')
-TEST_DATABASE_URL = os.environ.get('TEST_DATABASE_URL')
+if ENV == 'testing':
+    WAREHOUSE_URL = os.environ['TEST_WAREHOUSE_URL']
+    DATABASE_URL = os.environ['TEST_DATABASE_URL']
+else:
+    WAREHOUSE_URL = os.environ['WAREHOUSE_URL']
+    DATABASE_URL = os.environ['DATABASE_URL']
+    EXPLORERDB_URL = os.environ['EXPLORERDB_URL']
```

Comment:
Ah, never mind, I might have the "thinking out loud" syndrome...

Was thinking about a way to prevent accidents on explorer db – in this case – but trying to much to generalize it. Let's just forget about it 😅